spec/query_dsl_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe "QueryDSL" do
+  it "converts params to ES DSL" do
+    dsl = QueryDSL.new(params: {q: 'foo', aq: { color: "green" } }, current_user: nil)
+    expect(dsl.to_hash).to eq({
+      query: {
+        query_string: {
+          query: "(foo) AND (color:(green))",
+          default_operator: "and"
+        },
+      },
+      size: QueryDSL::MAX_RESULTS,
+      from: 0
+    })
+  end
+
+  # TODO pass with current_user and test filter
```

Comment:
ah ok, so perhaps this code comment would be more helpful as a comment in that Trello card so that whoever implements `current_user` knows to also update this file